zarr/store/zipfile.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+
+
+from zarr.store.base import ArrayStore
+
+
+class ZipFileStore(ArrayStore):
+
+    def __init__(self, path):
+        self._path = path
+        # TODO
+
+    @property
+    def meta(self):
+        pass
+
+    @property
+    def data(self):
+        pass
+
+    @property
+    def attrs(self):
+        pass
+
+    @property
+    def cbytes(self):
+        pass
+
+    @property
+    def initialized(self):
+        pass
```

Comment:
You mean reduce the entire ArrayStore interface to MutableMapping? I'm not sure that could be done as configuration metadata, user attributes and compressed chunk data all need to be managed in different ways potentially. Also an array store needs to implement the cbytes property and resize() method.